packages/ozone/src/daemon/config.ts

```diff
@@ -0,0 +1,88 @@
+import assert from 'assert'
+
+export interface DaemonConfigValues {
+  version: string
+  dbPostgresUrl: string
+  dbPostgresSchema?: string
+  serverDid: string
+  appviewUrl: string
+  appviewDid?: string
+  pdsUrl?: string
+  pdsDid?: string
+}
+
+export class DaemonConfig {
+  constructor(private cfg: DaemonConfigValues) {}
+
+  static readEnv(overrides?: Partial<DaemonConfigValues>) {
+    const version = process.env.BSKY_VERSION || '0.0.0'
+    const dbPostgresUrl =
+      overrides?.dbPostgresUrl || process.env.DB_PRIMARY_POSTGRES_URL
+    const dbPostgresSchema =
+      overrides?.dbPostgresSchema || process.env.DB_POSTGRES_SCHEMA
+    assert(dbPostgresUrl)
+    const serverDid = overrides?.serverDid || process.env.SERVER_DID
+    assert(serverDid)
+    const appviewUrl = overrides?.appviewUrl || process.env.APPVIEW_URL
+    assert(appviewUrl)
+    const appviewDid = process.env.APPVIEW_DID
+    const pdsUrl = process.env.PDS_URL
+    const pdsDid = process.env.PDS_DID
```

Comment:
Similar here, we might consider prefixing these with `OZONE_` for consistency with pds and recent appview work.